hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -2591,11 +2603,22 @@ public KeyProvider call() throws Exception {
     }
   }
 
+  @Override
+  public OzoneFsServerDefaults getServerDefaults() throws IOException {
```

Comment:
would be great to have a test to verify this where it makes sure a second call within the expiration time to this API does not invoke ozoneManagerClient.getServerDefaults()